### ISSUE TITLE
SegFault in tests on MacOS

### ISSUE THREAD

**Issue by neuroscr** [loki-project/loki-network#260]
make tests yeilds
```
[NFO] (681) Wed Jan 30 22:00:43 2019 PST llarp/link/server.cpp:179	Try establish to y8fiq3nr7a3nibe6na77apujexbdjxd9a19o9jie77oraexcuujy.snode
[NFO] (681) Wed Jan 30 22:00:43 2019 PST test/link/test_llarp_link.cpp:318	bob established with alice
[NFO] (681) Wed Jan 30 22:00:43 2019 PST test/link/test_llarp_link.cpp:280	alice established with bob
make: *** [test] Segmentation fault: 11
```

**Comment by neuroscr** [comment]
XCode tests don't crash but I get:
[0;31m[  FAILED  ] [m2 tests, listed below:
[0;31m[  FAILED  ] [mLinkLayerTest.TestUTPAliceRenegWithBob
[0;31m[  FAILED  ] [mLinkLayerTest.TestUTPAliceConnectToBob

**Comment by neuroscr** [comment]
should be fixed by #266 properly now
